Tools/jit/_targets.py

```diff
@@ -446,8 +446,7 @@ def _handle_relocation(
 def get_target(host: str) -> _COFF | _ELF | _MachO:
     """Build a _Target for the given host "triple" and options."""
     if re.fullmatch(r"aarch64-apple-darwin.*", host):
-        args = ["-mcmodel=large"]
```

Comment:
This was the start of the issues on Mac. `clang 18` really yells when you pass in this arg. I've tried passing this in as part of the args and it still seems to think that it's not being passed in. My hunch is that it may be impossible to disable on Mac and it's been forced on. In any case, removing it let me get further here for now.

```
clang: error: invalid argument '-mcmodel=large' only allowed with '-fno-pic'
```